apolo-sdk/src/apolo_sdk/_server_cfg.py

```diff
@@ -82,6 +82,12 @@ def key(self) -> Key:
         )
 
 
+@rewrite_module
+@dataclass(frozen=True)
+class AppsConfig:
+    apps_hostname_templates: Sequence[str] = ()
```

Comment:
I'm curious what is the meaning of this list?
What is the format for the list item?

Reply:
This is somewhat similar to `job_hostname_template`. This is a string with a placeholder `"{app_name}.apps.novoserve.org.neu.ro."`.  Also, Ivan mentioned, we might have a list of domain names for a single cluster in a future, therefore, it's a list of such placeholders.

End customers will not use it directly at the moment, therefore, I didn't add it to CLI itself.
We are going to use it in our apps deployment workflow, see https://linear.app/apolo/project/apps-support-various-base-domains-3e84552057e0/overview

Reply:
ok